Assets/MRTK/Examples/Demos/EyeTracking/General/Profiles/EyeTrackingDemoPointerProfile.asset

```diff
@@ -31,22 +31,27 @@ MonoBehaviour:
     handedness: 7
     pointerPrefab: {fileID: 1247086986094436, guid: 51e60b8742bc47640923ac9e75ea74e9,
       type: 3}
+    prioritizedLayerMasks: []
```

Comment:
@RogPodge it looks like pure reserialization doesn't update this properly, I think because we currently only set this in the custom inspector. I'm unvestigating using `ISerializationCallbackReceiver` instead

Reply:
😬 ouch, hopefully it's easily migratable.